### PR TITLE
[Fix] fix pytorch-paddle mapping index generate

### PR DESCRIPTION
**Pull request by RedContritio**
Fix display of [pytorch-paddle mapping index of release/2.6](https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.html) based on [develop version](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.html).

[Source PR](#6463) is approved and merged.
- #6463

> all new files have been merged to develop branch.

---

@jzhang533 @zhwesky2010 @sunzhongkai588 @luotao1

### CODE REVIEW

**Review by jzhang533 — APPROVED** [review]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.TripletMarginWithDistanceLoss.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+torch.nn.TripletMarginWithDistanceLoss(*, distance_function=None, margin=1.0, swap=False, reduction='mean')
+```
+
+### [paddle.nn.TripletMarginWithDistanceLoss](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/TripletMarginWithDistanceLoss_cn.html#tripletmarginwithdistanceloss)
+
+```python
+paddle.nn.TripletMarginWithDistanceLoss(distance_function=None, margin: float = 1.0, swap: bool = False, reduction: str = 'mean', name: str = None)
+```
+
+两者功能一致，参数完全一致。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.TripletMarginWithDistanceLoss.md:15]: 参数完全一致，也需要写一下 **参数映射表**

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.apply_.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## [ 参数完全一致 ]torch.Tensor.apply_
+### [torch.Tensor.apply_](https://pytorch.org/docs/stable/generated/torch.Tensor.apply_.html)
+
+```python
+torch.Tensor.apply_(callable)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.apply_.md:5]: 这个目前写了Matcher没

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool1d.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+torch.nn.AdaptiveAvgPool1d(output_size)
+```
+
+### [paddle.nn.AdaptiveAvgPool1D](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/AdaptiveAvgPool1D_cn.html#adaptiveavgpool1d)
+
+```python
+paddle.nn.AdaptiveAvgPool1D(output_size, name=None)
+```
+
+两者功能一致，参数完全一致。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool1d.md:15]: 参数完全一致，也需要写一下 参数映射表

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softplus.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+torch.nn.Softplus(beta=1, threshold=20)
+```
+
+### [paddle.nn.Softplus](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Softplus_cn.html)
+
+```python
+paddle.nn.Softplus(beta=1, threshold=20, name=None)
+```
+
+两者功能一致，参数完全一致。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softplus.md:15]: 参数完全一致，也需要写一下 参数映射表

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.slice_scatter.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+## [ 参数不一致 ]torch.slice_scatter
+
+### [torch.slice_scatter](https://pytorch.org/docs/stable/generated/torch.slice_scatter.html#torch.slice_scatter)
+
+```python
+torch.slice_scatter(input, src, dim=0, start=None, end=None, step=1)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.slice_scatter.md:6]: 这些是不是目前还没有写Matcher

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.TripletMarginWithDistanceLoss.md (hunk 1 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.TripletMarginWithDistanceLoss.md:15]: 这部分主要用于生成主目录时作为数据源，并不参与实际展示。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.apply_.md (hunk 2 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.apply_.md:5]: 尚未添加，文档存在就能修复 release/2.6 的问题
